_source/_posts/2021-10-16-full-stack-java.md

```diff
@@ -0,0 +1,693 @@
+---
+layout: blog_post
+title: "Full Stack Java with React, Spring Boot, and JHipster"
```

Comment:
the title is good, but I think we need to think what is this article about and put it in the title. Is the article about being a full stack java dev, or is it about creating an app with React, Spring Boot, and JHipster. Can only Java devs take something from this article, or maybe React or JS devs also can?

What do you think about this title:
Creating a Gallery app with React, Spring Boot, and JHipster.

Reply:
On the Okta developer blog, we try to stay away from "ing" words and would use something like "Create" instead. I'm not sure if Auth0 has similar suggestions. We also have a headline analyzer based on https://coschedule.com/headline-analyzer. 

- Creating a Gallery app with React, Spring Boot, and JHipster: 53
- Full Stack Java with React, Spring Boot, and JHipster: 59
- Full Stack Fun with React, Spring Boot, and JHipster: 63

I can give you access to the tool we use if you like. We also tend to optimize for SEO using Google's Keyword Planner. Here are some numbers about traffic worldwide:

`Term: avg monthly searches`
- react spring boot: 1300
- full stack java: 3600
- java full stack developer: 8100
- spring react: 18000

Should we optimize for 2-3 of these keywords?

Reply:
If we change the title, we should also change the URL to match, at least for this post that will (eventually) be syndicated on the OktaDev blog.

Reply:
Oh okay, if that's what you do in Okta blog, let's keep it this way ;)